websites/A/AI Dungeon/presence.ts

```diff
@@ -0,0 +1,224 @@
+const presence = new Presence({
+    clientId: "713563682722021436"
+});
+
+const browsingStamp = Math.floor(Date.now() / 1000);
+let login, register, target: string, bullsEye: string, playing, action;
+
+const check = window.addEventListener("click", function(event) {
+    target = (event.target as HTMLTextAreaElement).innerText;
+    if(target != undefined){
+        if(target.includes("Home")){
+            bullsEye = target;
+            return bullsEye;
+        }
+        else if(target.includes("Explore")){
+            bullsEye = target;
+            return bullsEye;
+        }
+        else if(target.includes("My Stuff")){
+            bullsEye = target;
+            return bullsEye;
+        }
+        else if(target.includes("Premium")){
+            bullsEye = target;
+            return bullsEye;
+        }
+        else if(target.includes("Contribute")){
+            bullsEye = target;
+            return bullsEye;
+        }
+        else if(target.includes("Settings")){
+            bullsEye = target;
+            return bullsEye;
+        }
+        else if(target.includes("About")){
+            bullsEye = target;
+            return bullsEye;
+        }
+        else if(target.includes("New Updates")){
+            bullsEye = target;
+            return bullsEye;
+        }
+        else if(target.includes("NEW SINGLEPLAYER GAME") || target.includes("CONTINUE GAME") || target.includes("NEW MULTIPLAYER GAME")){
+            bullsEye = target;
+            return bullsEye;
+        }
+        else if(target.includes("")){
+            return bullsEye = "NEW SINGLEPLAYER GAME";
+        }
+        else{
+            return bullsEye;
+        }
+    }
+    else{
+        return bullsEye = "Home";
+    }
+});
+presence.on("UpdateData", async () => {
+    const presenceData: PresenceData = {
+        largeImageKey: "ailogo"
+    };
+    // Due to unfortunate errors regarding regExp the main site aidungeon.io is pushed back to a later date.
+    // if(window.location.hostname == "aidungeon.io"){
```

Comment:
Try: `([a-z0-9-]+[.])*aidungeon[.]io[/]`

Reply:
That fixes it! Thank you.